cluster-baseline-settings/akv-secrets-store-csi.yaml

```diff
@@ -194,7 +335,7 @@ spec:
         app.kubernetes.io/component: csi-driver
     spec:
       nodeSelector:
-        beta.kubernetes.io/os: linux
+        kubernetes.io/os: linux
```

Comment:
There are 2 instances using `beta.kubernetes.io/os` left in each [cluster-baseline-settings/flux.yaml](cluster-baseline-settings/flux.yaml) and [cluster-baseline-settings/aad-pod-identity/aad-pod-identity.yaml](cluster-baseline-settings/aad-pod-identity/aad-pod-identity.yaml).  Should these be changed for consistency?

Reply:
yeah, good catch.  I'll update those.